pkg/controller/types.go

```diff
@@ -29,6 +29,7 @@ const (
 	KubernetesLabelAppComponent = "app.kubernetes.io/component"
 	KubernetesLabelAppPartOf    = "app.kubernetes.io/part-of"
 	KubernetesLabelAppManagedBy = "app.kubernetes.io/managed-by"
+	KubernetesLabelCreatedBy    = "app.kubernetes.io/created-by"
```

Comment:
nitpick: this label is deprecated https://kubernetes.io/docs/reference/labels-annotations-taints/#app-kubernetes-io-created-by-deprecated